web/src/components/layout/layout.tsx

```diff
@@ -180,9 +198,17 @@ class Layout extends React.PureComponent<LayoutProps, LayoutState> {
       'staging-banner-is-visible': showStagingBanner,
     });
 
+    const alreadyEnrolled =
+      this.state.showWelcomeModal &&
+      user.account &&
+      user.account.enrollment &&
+      user.account.enrollment.challenge;
```

Comment:
This will be so much nicer once we upgrade typescript!

`user.account?.enrollment?.challenge`

...siiiigh, someday! 😍